napari/utils/action_manager.py

```diff
@@ -159,7 +159,7 @@ def __init__(self):
         self._buttons: Dict[
             str, Set[Tuple(Union[QPushButton, QtStateButton], str)]
         ] = defaultdict(lambda: set())
-        self._shortcuts: Dict[str, str] = {}
+        self._shortcuts: Dict[str, list[str]] = defaultdict(lambda: set())
```

Comment:
Set does not preserve order, should it be a list (like the type definition?

Reply:
yeah I'd really like a set as if something is present twice it will really mess things up. I'll change to set on the LHS.